Omulosi/iReporter
# #162660020 Fix delete endpoint

#### What does this PR do?
#### Description of Task to be completed?
* Query database for the ID of the record to be deleted and check if it exists. Return an error if it does not otherwise delete the record
#### How should this be manually tested?
clone the repo and checkout the bg-fix-delete-endpoint-162660020. Run the application as documented in the README. Create a user and create an intervention record. Deleting the record should be successful. Deleting a record with an invalid ID should result in a descriptive error response.
#### Any background context you want to provide?
The changes are in response to a feedback review
#### What are the relevant pivotal tracker stories?
#162660020
